### ISSUE TITLE
US_3.16: Personal Emails

### ISSUE THREAD

**Issue by tobiduer** [hpi-swt2/workshop-portal#58]
**As**
applicant

**I want to**
receive emails with my personal name in the salutation

**in order to**
see that each individual is important to the HPI.

**Comment by Baschdl** [comment]
@tobiduer Should personal pronouns etc. also be adapted?

**Comment by tobiduer** [comment]
@Baschdl: No. The author has to adjust the text to fix the second person singular.
